marlowe-dashboard-client/src/Marlowe/README.md

```diff
@@ -0,0 +1,38 @@
+# About Marlowe Contracts and the PAB
```

Comment:
Awesome notes 🙌

I do wonder if we should better move this inside of `marlowe/PAB.purs` to be next with the types.

Reply:
I started trying to put some comments in there, and it was hard to explain it well, since everything is related. So I thought a separate README would be better. Maybe not though... Still, going to leave it as it is for now.